crates/extensions/aws_mapper_ext/src/converter.rs

```diff
@@ -227,31 +218,12 @@ mod tests {
             "time": "2021-04-08T00:00:00+05:00"
         });
 
-        let output = converter.convert(&new_tedge_message(input)).await;
+        let output = converter.convert(&new_tedge_message(input)).unwrap();
 
         assert_json_eq!(
             serde_json::from_str::<serde_json::Value>(&extract_first_message_payload(output))
                 .unwrap(),
             expected_output
         );
     }
-
-    #[tokio::test]
-    async fn exceeding_threshold_returns_error() {
```

Comment:
Removed this test because the creation of converter does not accept the `threshold` value any more.

Reply:
But there is still a threshold check. Isn't it?

So I would not remove that test, adding a `with_threshold(threshold: usize)` method if necessary.